src/commands/purge.ts

```diff
@@ -8,7 +8,7 @@ module.exports = new Command({
 	props: [{ name: 'number', required: false }],
 	async execute({ ia }) {
 
-		const num = ia.options.data[0].value ? Number.parseInt(ia.options.data[0].value) : 100
+		const num = ia.options.data[0].value ? Number.parseInt(ia.options.data[0].value.toString()) : 100
```

Comment:
why did you not use options.get... here? if there was no reason you could use ia.options.getNumber('number') || 100

Reply:
Well we're not passing the props through at the moment, so I'm assuming it won't return anything under that named slash option? Whereas `data` is supposed to contain an array of the passed params so it ought to be there as the first param :)

Reply:
Also just converted it over from what where there, it's the same format as before as I don't get what the `x = something ? something : something` structure is doing lol

Reply:
it's a if ... else shorthand. It's probably best to create the options first then. I'll try it tomorrow, but I don't wanna change all commands again

Reply:
Oooh it's just if/else, some more changes to do then.. :)

Reply:
Yeah I would prefer to implement the options building and then use option.get everywhere

Reply:
If you can build it in somehow, happy to change it all over! :D